apps/jenkins/src/main/fabric8/openshift-deployment.yml

```diff
@@ -76,6 +76,8 @@ spec:
             secretKeyRef:
               name: jenkins-recommender-api-token
               key: token
+        - name: JENKINS_ROOT_URL
+          value: "${JENKINS_ROOT_URL}"
```

Comment:
As @vpavlin said prod-preview has (https://jenkins.prod-preview.openshift.io) and production (https://jenkins.openshift.io) - URL's?

To achieve this following could be possible options:
1) Templating: To use `${JENKINS_ROOT_URL}` template variable CI/CD or deployment tools can substitute this value
e.g. 
`oc <> -P JENKINS_ROOT_URL=https://jenkins.openshift.io`
`mvn fabric8:resource -DJENKINS_ROOT_URL=https://jenkins.openshift.io`

2) Use Configmaps?

 @rupalibehera @aslakknutsen

Reply:
@hrishin 1 and 2 are the same. The 'deployment tool' in question here is the Tenant Service. There is no ConfigMap outside of what Tenant Service sets up for you. If you want to set it up via the "jenkins" ConfigMap(https://github.com/fabric8-services/fabric8-tenant-jenkins/blob/master/apps/jenkins/src/main/fabric8/openshift-cm.yml) or directly on the DeploymentConfig is same same for me. But the 'real' value will be in the Tenant Service ConfigMap.